### ISSUE TITLE
Loading Spinner: wrong behaviour

### ISSUE THREAD

**Issue by JohnHavok** [SAP/fundamental-ngx#1272]
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
LoadingSpinnerComponent.loading doesn't work with fundamental-style 0.2.0 and his two middle bars are missing.

#### Which versions of Angular and Fundamental NGX are affected? (If this is a feature request, use current version.)
Angular 7 and Fundamental NGX 0.10.0

#### If this is a bug, please provide steps for reproducing it.

create a modal *with fundamental-style version 0.2.0*

#### Please provide relevant source code if applicable.
<details>
<summary> code </summary>

```html
<div [attr.aria-busy]="translationPage?.content == undefined">
  <fd-loading-spinner [loading]="translationPage?.content == undefined"></fd-loading-spinner>
  <app-table></app-table>
</div>
```
</details>

![image](https://user-images.githubusercontent.com/34537974/64949711-19161c80-d87a-11e9-89f1-a49dd15252be.png)
The expected behavior in this image was a hidden loading-spinner but he's not hidden and he's missing his two middle bars ( _fd-loading-spinner_ have no child _div_ to have class .fd-spinner__body ? )

#### Is there anything else we should know?
Maybe it's a ticket for fundamental-style, if that's the case. My apologize...

**Comment by JKMarkowski** [comment]
Hi @JohnHavok, the fix will be pushed with fundamental-styles version 0.2.0 adaptation.

**Comment by JKMarkowski** [comment]
Hi @JohnHavok It's solved in 0.12.0 version.
